README.md

```diff
@@ -1,2 +1,120 @@
+                                                    text                                             tokens
+index                                                                                                      
+12313  Phylogenetic networks are a generalization of ...  [Phylogenetic, networks, are, a, generalizatio...
+3172   Prediction modelling is more closely aligned w...  [Prediction, modelling, is, more, closely, ali...
+6451   The heat transfer exhibits the flow of heat (t...  [The, heat, transfer, exhibits, the, flow, of,...
+4351   a common experience during superficial ultraso...  [a, common, experience, during, superficial, u...
+22694  Code metadata Current code version v1.5.9 Perm...  [Code, metadata, Current, code, version, v1.5....
+```
+
+## Evaluation
+
+We're using Macro F1 score on `token_label_ids`. For each Full text, we're tokenizing/splitting the text on whitespace, and labeling each token. The final score is the average f1 across all full texts in the test set.
+
+You can test your solution's performance offline by using the provided evaluation script [eval_f1.py](src/eval_f1.py)
+
+Usage:
+```
+poetry run python -m src.eval_f1 --true_labels_file <file-in-data-dir-with-true-labels>.parquet --pred_file predictions.parquet
+```
+
+> Both files (true labels file and predictions file) must be located in [data](data)
+
+### To submit a solution:
+
+1. Prepare a Parquet file **predictions.parquet**(filename matters) with index "index" and a column "preds" structured as follows:
+
+index      preds
+14104    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ...
+21468    [3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, ...
+The length of a list "preds" should match the corresponding length of a "tokens" list in the dev/test set (i.e., one prediction per token).
+
+Here preds is a list of integers from 0 to 3 corresponding to token-level predictions:
+
+- 0 for 'human'
+- 1 for 'NLTK_synonym_replacement'
+- 2 for 'chatgpt'
+- 3 for 'summarized'
+
+See "[sample_dev_submission.zip](https://drive.google.com/file/d/1Xm59UlLJ-aemDG-PZ3SCBJa_5EkzGLXq)" (from the [Data page](https://www.codabench.org/competitions/2431/#/pages-tab)) for an example. It is an all-zero submission that corresponds to predicting that all text is human-written.
+
+2. Compress (zip) the prediction file, for example, with a bash command:
+
+zip my_new_submission.zip predictions.parquet
+
+Please note that you should not compress the folder with the prediction files. Instead, you need to compress the files themselves: i.e., if you are submitting from Macbook (and you do not want to compress via terminal), you need to select all the prediction files that you want to send and press "Compress".
+
+3. Submit your solution via the My submissions tab.
+
+## Baseline
+
+We're also providing a baseline where we use [DistilBERT](https://huggingface.co/docs/transformers/model_doc/distilbert) to approach the task as an NER task. To run the baseline, execute:
```

Comment:
```suggestion
We're also providing a baseline where we use [DistilBERT](https://huggingface.co/docs/transformers/model_doc/distilbert) to approach the task as an NER task. To run the baseline, execute:
```

We're also providing a baseline where we use [DistilBERT](https://huggingface.co/docs/transformers/model_doc/distilbert) to approach the task as a token classification task. To run the baseline, execute: